Makefile

```diff
@@ -23,7 +23,7 @@ MACHINE_IP = $(shell hostname -I | cut -d' ' -f1)
 
 include repo.config
 
-
+SERVICES := adminer deployment-agent graylog maintenance minio monitoring portainer portus traefik

```

Comment:
-  Why not dynamically deducing the service names using ``docker-compose config --services`` or simply listing all folders under services, e.g. see make's [wildcard function](https://www.gnu.org/software/make/manual/make.html#Wildcard-Function)??

Reply:
done